packages/office-ui-fabric-react/src/components/Conformance.test.tsx

```diff
@@ -0,0 +1,212 @@
+//        of adding it to the exclusion list as this will make regression harder to detect.
+const excludedComponents: string[] = [
+  'Beak', // className is not injected
+  'Button', // deprecated, test Button variants instead
+  'ChoiceGroupOption', // className is not injected
+  'Coachmark', // className is not injected
+  'ColorRectangle', // className is not injected
+  'ContextualMenuItemWrapper', // className is not injected
+  'Dialog', // className is deprecated
+  'HoverCard', // className is not injected
+  'Keytip', // helper component, not meant to take a className
+  'KeytipData', // helper component, not meant to take a className
+  'KeytipLayer', // helper component, not meant to take a className
+  'Layer', // className is not injected
+  'PersonaPresence', // className is not injected
+  'PositioningContainer', // className is not injected
+  'Rating', // className is not injected
+  'SelectedPeopleList', // does not accept className
+  'ShimmerCircle', // className is not injected
+  'ShimmerElementsGroup', // className is not injected
+  'ShimmerLine', // className is not injected
+  'ShimmerGap', // className is not injected
+  'SpinButton', // className is not injected
+  'Sticky', // accepts stickyClassName instead of className
+  'TeachingBubble', // does not accept className
+  'ThemeGenerator' // not intended to be tested
+];
+
+// Some components require nodes to be mocked when creating the test component (e.g. components that use refs)
+const mockNodeComponents = ['ScrollablePane'];
+
+/**
+ * Automatically consume and test any components that are exported
+ * from a file that matches its parent folder name (e.g. ActivityItem/ActivityItem.tsx)
+ *
+ * If you are here and are getting a failure due to a component you recently added,
+ *    here are some options:
+ *
+ *    1) Is the test failing due to a runtime error in the component, or is the component null?
+ *       The component may require certain props in order to render. Consider adding the props to the
+ *       requiredProps object above.
+ *
+ *    2) Does the component inject its className prop onto an element other than the root?
+ *       Consider adding the class name of the element where the className prop is injected
+ *       to the classNameSelectors list above.
+ *
+ *    3) Does the component use refs? It may need DOM nodes to be mocked.
+ *       Consider adding the component name to the mockNodeComponents list above.
+ *
+ * In any case, adding your component to the exclusion list is discouraged as this will make regression
+ *    harder to catch.
+ */
+describe('Conformance', () => {
+  beforeAll(() => {
+    // Mock Layer since otherwise components that use Layer will have empty JSON representations
+    jest.mock('./Layer', () => {
+      return {
+        Layer: jest.fn().mockImplementation(props => {
+          return props.children;
+        })
+      };
+    });
+  });
+
+  const glob = require('glob');
```

Comment:
You should move glob and path to the first 2 lines (as an import, rather than require).